iceoryx_posh/include/iceoryx_posh/internal/popo/gateway/gateway_generic.inl

```diff
@@ -12,36 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef IOX_DDS_INTERNAL_GATEWAY_DDS_GATEWAY_GENERIC_INL
-#define IOX_DDS_INTERNAL_GATEWAY_DDS_GATEWAY_GENERIC_INL
+#ifndef IOX_POSH_GATEWAY_GENERIC_INL
+#define IOX_POSH_GATEWAY_GENERIC_INL
```

Comment:
`POPO` is missing (`IOX_POSH_POPO_GATEWAY_GENERIC_INL`).
See `building_blocks` for reference.
In case it stays in namespace `popo`.